preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -192,7 +192,7 @@ def enrich_with_nextclade(  # noqa: C901, PLR0912, PLR0914, PLR0915
             elif len(unaligned_segment) == 1:
                 num_valid_segments += 1
                 unaligned_nucleotide_sequences[id][segment] = (
-                    entry.data.unalignedNucleotideSequences[unaligned_segment[0]]
+                    trim_ns(entry.data.unalignedNucleotideSequences[unaligned_segment[0]])
```

Comment:
@anna-parker - Q: will this work for the non-nextclade case too? If not do you mind figuring out how we can since you know the code better?

Reply:
Oh no good catch this will in fact not work for non-nextclade cases! (your approach would though so if need be we can combine them)

Reply:
Maybe actually we don't need to worry too much about that case since it's a light touch pipeline in that case anyway

Reply:
Guess we might as well fix this: https://github.com/loculus-project/loculus/pull/4081